### ISSUE TITLE
Missing buffs

### ISSUE THREAD

**Issue by ratosh** [UltraMachine/rust-sc2#50]
From discord discussion the lib is missing those buffs:
```
	SunderingImpact = 299,
	AmplifiedShielding = 300,
	PsionicAmplifiers = 301,
	SecretedCoating = 302,
	OnCreepVisible = 303,
```

**Comment by UltraMachine** [comment]
fix with 5bcde2d

**Comment by ratosh** [comment]
Can we have a new version with this buff? Library panics when a map is using a unknown buff.
